src/index.js

```diff
@@ -5,6 +5,7 @@ import MdBottomBar from './components/mdBottomBar';
 import MdButton from './components/mdButton';
 import MdButtonToggle from './components/mdButtonToggle';
 import MdCard from './components/mdCard';
+import MdChips from './components/mdChips';
```

Comment:
My push of the chips is also merged. I'll revert the `367eea8` commit so the repo only contains the sample for switch.